src/qt/utils.cpp

```diff
@@ -100,7 +114,6 @@ void registerXdgMime(QApplication &app){
 
     if (TailsOS::detect() && TailsOS::detectDotPersistence() && TailsOS::usePersistence) {
         TailsOS::persistXdgMime(filePath, mime);
-        return;
```

Comment:
?

Reply:
desktop file and icon should also be written to default locations on Tails.